readyplayerme/rpm-unity-sdk-avatar-creator
# [4221706127] Add user avatars selection

## [4221706127](https://ready-player-me.monday.com/boards/2563815861/pulses/4221706127)

## Description

-  Added user avatar screen which shows all created avatars by logged in user, they can be customized or selected




## Changes

#### Added

- User avatar selection screen
- API for fetching all user avatars
- API for getting the pre-created avatar gameobject
- Background image for selection screens

#### Updated

- Fix for refresh session not getting stored
- Refactored profile manager 



## How to Test

- Login -> it should take you to avatar selection
- Try customize and select an avatar



## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
